Sources/Prometheus/RELEASE.md

```diff
@@ -0,0 +1,50 @@
+# Releasing 
+
+## Release schedule
+
+Releases of this library are performed on a when-needed basis. 
+
+Generally one of the maintainers listed in MAINTAINERS.md will perform a release.
+
+If you would like to request a release since a change or fix has been waiting for a release for some time,
+you may open up an issue on github, requesting a release.
+
+## Versioning 
+
+As usual with Swift packages, we use [Semantic Versioning](https://semver.org/).
+
+## Make a release
+
+Development and releases are made from the `main` branch directly, therefore the branch should remain in releasable state at any time.
```

Comment:
Just a comment: It’s the current state of things. We might want to consider forming the habit of creating a release or branch.

Reply:
For larger projects we might, but I'd be wary of introducing overhead for libs which we revisit rarely, like this one 👀